### ISSUE TITLE
Porting to windows

### ISSUE THREAD

**Issue by LaggAt** [fukawi2/check_urbackup#1]
Hi, I'm about porting your check to windows. I'll basically rewrite the code in python as unix-scripts will not run on windows :)

May I ask you for a pull request later?

edit: maybe I'll use something other than python, as the client-installation should be easier.

**Comment by fukawi2** [comment]
Hi, glad you've found this useful enough to consider rewriting :)
I'm actually not using urbackup anymore, so I'm not maintaining this script
(and not planning to in the near future) so it is perhaps best if you just
create a new project and use this as reference?

**Comment by LaggAt** [comment]
Well, I searched a cheap and easy backup solution for my father, so this will be a small holiday project. UrBackup seems to be easy without much configuration, but I need to get informed if it's failing anytime in future. Therefore I will add it to my Icinga2 instance, where I need a check like this.

I currently also consider if I will use it for my own machines, where I would backup to a Linux based server. So I could test both checks, maybe I can use a single C++ code base for both in a new project.

You may follow the progresss in my fork, feel free to pull it back whenever you like.
https://github.com/LaggAt/check_urbackup

May I ask why you switched to which solution? Eventually there's a better solution?

**Comment by fukawi2** [comment]
I moved away at work because we were having issues getting all the clients talking to the server when they weren't on the same subnet -- it was a while ago now but it was taking too much of our time to debug.

I moved away from it at my parents retail store because I moved them from a HP Microserver to a Synology NAS so it was easier to just run the free Veeam endpoint product pointed at the NAS rather than having to manage urBackup/run it on the limited NAS hardware.

I still think it's a great project and will definitely reevaluate it in the future once it matures for other environments. I certainly do not discourage anyone trailing/using it. It just didn't fit my use-case at the time. Best of luck :)
